experimenter/experimenter/targeting/constants.py

```diff
@@ -211,6 +211,33 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEED_DEFAULT_NEED_PIN_WINDOWS_1903 = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default & pin"
+    ),
+    slug="first_run_need_default_need_pin",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default & pin"
+    ),
+    targeting=(
+        "{first_run} && os.windowsBuildNumber >= 18362 && {new_profile} && "
```

Comment:
Should we reuse `
FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903` similar to how its done in `FIRST_RUN_NEW_PROFILE_HAS_PIN_NEED_DEFAULT_WINDOWS_1903`, in this case we can use doesAppNeedPin directly

Reply:
Done! Also move the targeting down so that it's next to the related ones